config/config.js

```diff
@@ -55,6 +56,14 @@ var config = {
                 "angborxley",
                 "prune2000",
             ],
+            "heltour": {
+                "token": heltour_token,
+                // TODO: this should be pulled from somewhere else so that
+                // config doesn't require updating each time a season changes
+                "base_endpoint": "https://www.lichess4545.com/api/", 
```

Comment:
do you mean to refer to this line with the comment above? or just the season below it?